tests/test_association_db_sqlite.py

```diff
@@ -384,6 +384,60 @@ def test_load_dia_sources(self):
         empty_dia_sources = self.assoc_db.load_dia_sources([6])
         self.assertEqual(len(empty_dia_sources), 0)
 
+    def test_store_dia_sources_different_schema(self):
+        """Test the storage of DIASources in the database.
+        """
+        # Create test associated DIAObjects and DIASources.
+        schema = afwTable.SourceTable.makeMinimalSchema()
+        schema.addField('base_PsfFlux_flux', type='D')
+        schema.addField('base_PsfFlux_fluxSigma', type='D')
+        schema.addField('junk1', type='L')
+        schema.addField('junk2', type='D')
+        schema.addField('junk3', type='L')
+        schema.addField('junk4', type='D')
```

Comment:
I'm not sure I get the point of the test. Is it that `schema` doesn't need `psFlux` but can ignore unknown columns?

Reply:
Exactly, I'll add comments to that effect.